cli/pcluster/cli.py

```diff
@@ -155,9 +155,11 @@ def _get_parser():
     pcreate.add_argument(
         "-u",
         "--template-url",
-        help="Specifies the URL for a custom CloudFormation template, " "if it was used at creation time.",
+        help="Specifies the URL for a custom CloudFormation template, if it was used at creation time.",
+    )
+    pcreate.add_argument(
+        "-t", "--cluster-template", help="Indicates the cluster section of the configuration file to use."
```

Comment:
"Indicates which section of the configuration file to use for cluster creation."